test/asynchronous/__init__.py

```diff
@@ -861,6 +861,16 @@ async def max_message_size_bytes(self):
 async_client_context = AsyncClientContext()
 
 
+async def recreate_client_context():
+    if _IS_SYNC:
+        # sync tests don't need to recreate a client context
+        return
+    global async_client_context
+    await async_teardown()
+    async_client_context = AsyncClientContext()
```

Comment:
This doesn't work because the old `client_context` was already imported by other files so this change won't be reflected there. For example:
```python
>>> import pymongo
>>> from pymongo import ASCENDING
>>> ASCENDING
1
>>> pymongo.ASCENDING = 2
>>> ASCENDING
1
>>> pymongo.ASCENDING
2
```

Instead we need to mutate the existing ClientContext

Reply:
Ahhh that makes sense, thanks!